ivandocs/ACS_Places_push.md

```diff
@@ -0,0 +1,99 @@
+# Adobe Campaign Standard
+## Push Notifications with Experience Platform Location Service
+
+In this guide, we’ll show how you can use historical geo-location information to target push notifications delivered through Adobe Campaign Standard. 
+
+Before we begin, this guide assumes that you have a mobile application already configured with the Adobe Experience Platform Mobile SDK, including the Adobe Campaign Standard extension. 
+
+### Prerequisites
+
+1. Integrate the [Adobe Experience Platform Mobile SDK](https://aep-sdks.gitbook.io/docs/getting-started/get-the-sdk) into your app.
+2. Add the [Adobe Campaign Standard Extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaign-standard) to your mobile app configuration.
+3. [Create one or more POIs](https://placesdocs.com/places-services-by-adobe-documentation/places-database-management-1/managing-pois-in-the-places-ui#create-a-poi) in Adobe Experience Platform Location Services.
+4. Enable and install the [Adobe Places extensions for Adobe Experience Platform Mobile SDKs]()
+
+
+## Create data elements in Experience Platform Launch
+
+After you verify that the Places and Places Monitor for Location Service extensions are working correctly in your application, create data elements in Experience Platform Launch. Data elements allow you to read the information that was provided by the extensions coming through the Mobile SDK event hub and act as an alias to retrieve data from the client application. Let’s create a few data elements to retrieve data from the Places extensions, so that we can send the Places information to Campaign.
+
+1. In of your Experience Platform Launch mobile property, click the **Data Elements** tab and click **Add Data Element**.
+2. In the **Extension** drop-down list, select **Places**.
+3. From the **Data Element Type** drop-down list, select **Name**.
+4. In the right-hand side pane, you can select **Current POI** which retrieves the name of the POI in which the user is currently located.
+
+   **Last Entered** retrieves the name of the POI that user last entered, and **Last Exited** provides the name of the POI that the user last left. For this example, we will selected **Last Entered** and type a name for the data element, such as **Last Entered POI Name** and clicked **Save**.
+
+![](images/ACS_Push1.png)
+
+5. Repeat the same steps above and create data elements for _Last Entered POI Latitude_, _Last Entered POI Longitude_, and _Last Entered POI Radius_.
+
+In addition to the data elements for Location Service, ensure that you create Mobile Core data elements for _App ID_ and _Experience Cloud ID_.
+
+## Create a rule to send location data to Adobe Campaign Standard
```

Comment:
This information is also provided by @sbenedicadb  at
 https://github.com/jiabingeng/places-service-docs/blob/master/places-solution-integrations/campaign/postbacks.md


we might need to consolidate them in one